misk/src/test/kotlin/misk/web/exceptions/ExceptionMapperTest.kt

```diff
@@ -70,15 +70,15 @@ internal class ExceptionMapperTest {
   class ThrowsActionException @Inject constructor() : WebAction {
     @Get("/throws/action/{statusCode}")
     @ResponseContentType(MediaTypes.TEXT_PLAIN_UTF8)
-    fun throwsActionException(@PathParam statusCode: StatusCode): Nothing {
+    fun throwsActionException(@PathParam statusCode: StatusCode): String {
       throw ActionException(statusCode, "you asked for an error")
     }
   }
 
   class ThrowsUnmappedError @Inject constructor() : WebAction {
     @Get("/throws/unmapped-error")
     @ResponseContentType(MediaTypes.TEXT_PLAIN_UTF8)
-    fun throwsUnmappedException() {
+    fun throwsUnmappedException() : String {
```

Comment:
little improvement: all actions need to have return values now otherwise the application refuses to boot